src/notation/view/notationnavigator.h

```diff
@@ -76,16 +96,16 @@ class NotationNavigator : public AbstractNotationPaintView
     void mousePressEvent(QMouseEvent* event) override;
     void mouseMoveEvent(QMouseEvent* event) override;
 
-    void paintCursor(QPainter* painter);
     void paintPageNumbers(QPainter* painter);
 
-    void moveCanvasToRect(const RectF& viewRect);
+    bool moveCanvasToRect(const RectF& viewRect);
 
     bool isVerticalOrientation() const;
 
     PageList pages() const;
 
     RectF m_cursorRect;
+    std::unique_ptr<NotationNavigatorViewRect> m_viewRect;
```

Comment:
Would it be an idea to rename `m_viewRect` to `m_cursorRectView`, to make super clear what's the relation with `m_cursorRect`?